src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = (int) side;
+    }
+
+    public Square(String color) {
```

Comment:
If you intend to provide a default value for the 'side' when using the constructor on line 11, you should initialize it here with a constant value for better clarity and to avoid having an uninitialized 'side' which could lead to issues when calling 'getArea' on such an instance.